src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/index.js

```diff
@@ -42,8 +42,7 @@ const styles = css`
     display: inline-block;
     font-size: 22px;
     font-weight: 500;
-    line-height: 1.8;
-    vertical-align: bottom;
+    line-height: 1.2;
```

Comment:
Do the same for `BaggageInfo/index.js` line 39

Reply:
Done